conf/application.conf

```diff
@@ -105,11 +105,13 @@ frontend-audit {
   masked-form-fields = "password,authCode,ggPassword"
 }
 
-google-analytics {
-  token = N/A
-  host = auto
+timeout {
+    timeout = 900
+    countdown = 120
 }
 
+timedOutUrl = "http://localhost:9233/payrollbik/we-signed-you-out"
```

Comment:
this is not required if we do change signout link to @routes.SignedOutController.signedOut() in main_template